membership-attribute-service/app/controllers/AccountController.scala

```diff
@@ -98,7 +98,11 @@ class AccountController(commonActions: CommonActions, override val controllerCom
       val cancellationSteps = for {
         _ <- EitherT(tp.zuoraRestService.disableAutoPay(zuoraSubscription.accountId)).leftMap(message => s"Error while trying to disable AutoPay: $message")
         _ <- EitherT(tp.zuoraRestService.updateCancellationReason(zuoraSubscription.name, reason)).leftMap(message => s"Error while updating cancellation reason: $message")
-        cancelResult <- EitherT(tp.zuoraRestService.cancelSubscription(zuoraSubscription.name)).leftMap(message => s"Error while cancelling subscription: $message")
+        subHasBeenInvoiced = zuoraSubscription.plans.list.exists{
```

Comment:
I welcome any suggestions on how I could invert this, such that the variable could be `subHasNotBeenInvoiced` so there is no negation on line 105